spec/classroom_spec.rb

```diff
@@ -0,0 +1,12 @@
+require './spec_helper' # Adjust the file path as needed
+
+describe Classroom do
+  let(:classroom) { Classroom.new('Class A') }
+  let(:student) { double('Student') }
+
+  it 'should have a label and an empty list of students' do
+    expect(classroom.label).to eq('Class A')
+    expect(classroom.students).to be_an(Array)
+    expect(classroom.students).to be_empty
+  end
+end
```

Comment:
- [ ] It would be nice if you could add a test for the add_student method in the classroom class as required. It's required that you write tests for all the methods in all the classes in their separate test file. Kindly add that 👍🏽